### ISSUE TITLE
Error: "The name Name already exists" in Lab 2, Steps 11 & 12 of Learning Path 02

### ISSUE THREAD

**Issue by usamabinrauf** [MicrosoftLearning/MB-500-Microsoft-Dynamics-365-Finance-and-Operations-Apps-Developer#78]
# Module: Learning Path 02: Build finance and operations apps
### Description of issue:

When entering the value **Name** in the **Data entity field name** and **Label Id** fields, I get the "**The name Name already exists**" error.

Upon investigation, it appears that the field named **Party** already uses the value **Name** for its **Data entity field name** and **Label** fields. This conflict prevents the **Name** value from being reused in Steps 11 and 12.

Screenshots of the error and the conflicting field configuration have been provided below for reference.

### Screenshots:

![Screenshot_3](https://github.com/user-attachments/assets/6957c608-c74b-4e28-b659-c7ad4c0ed4b6)
![Screenshot_1](https://github.com/user-attachments/assets/d7d8edb3-b72d-4994-bf48-aabffd2576a7)
![Screenshot_2](https://github.com/user-attachments/assets/a184754f-4890-43e6-b81c-3505acea62c5)

**Comment by janeteblake** [comment]
Thank you so much for reporting this! We will investigate and get back to you.
Janet

**Comment by janeteblake** [comment]
Hello Usama, we have tested this lab. Multiple lab testers have not been able to recreate the issue. I investigated, and noticed that your screenshot (thanks for the clear screenshots!) shows the checkbox "Convert labels to field names" checked whereas our VMs did not have that checked. 

However, we then reran the lab with that box incorrectly checked, and did not recreate your error.

What is your VM version and Authorized lab hoster? Currently we are working with 10.0.0.37.
Thank you,
Janet
